app/javascript/src/js/formValidations.js

```diff
@@ -1,121 +1,39 @@
 /* eslint-disable */
 
-import validate from 'validate.js';
-
-const constraints = {
-  'user[username]': {
-    length: {
-      minimum: 4,
-      maximum: 20,
-    },
-  },
-  'user[email]': {
-    email: true,
-  },
-  'user[password]': {
-    length: {
-      minimum: 6,
-      maximum: 128,
-    },
-  },
-  'user[password_confirmation]': {
-    equality: {
-      attribute: 'user[password]',
-      message: '^The passwords do not match',
-    },
-  },
-  'user[current_password]': {
-    length: {
-      minimum: 6,
-      maximum: 128,
-    },
-  },
-  'user[learning_goal]': {
-    length: {
+import { Application } from "stimulus";
+import { ValidationController } from "stimulus-validation"
+// https://github.com/jwald1/stimulus-validation
+
+export default class SignupValidation extends ValidationController {
```

Comment:
just to make this more generic and reusable can it be renamed to:
```suggestion
export default class FormValidation extends ValidationController {
```